playbooks/provisioning/openstack/post-provision-openstack.yml

```diff
@@ -59,12 +59,20 @@
 
 - name: OpenShift Pre-Requisites
   hosts: OSEv3
-  gather_facts: False
+  gather_facts: true
   become: true
-  tasks:
+  pre_tasks:
   - name: "Include DNS configuration to ensure proper name resolution"
     lineinfile:
       state: present
       dest: /etc/sysconfig/network
       regexp: "IP4_NAMESERVERS={{ hostvars['localhost'].private_dns_server }}"
       line: "IP4_NAMESERVERS={{ hostvars['localhost'].private_dns_server }}"
+  roles:
+  # CentOS isn't using NetworkManager by default. Make sure it's installed:
+  - role: node-network-manager
```

Comment:
I wrote it so from this pull request. It's basically the same thing as the network_manager.yml plabook does in openshift-ansible but turned into a role.

Theoretically, we could move the role there and make sure both playbooks use it, but it's tiny so I'm not sure it matters.